.github/workflows/dsm-test.yml

```
@@ -86,6 +86,7 @@ jobs:
         run: cd akeneo-design-system/example && yarn install
 
       - name: Launch example integration tests
+        id: example_integration_tests
         run: cd akeneo-design-system/example && yarn test
```

I not sure about the added line on visual_test_execution but should we add :
```suggestion
        run: cd akeneo-design-system/example && yarn test && echo ::set-output name=status::success
```

---

yes